scripts/generateAssetData/arbitrum/index.ts

```diff
@@ -1,13 +1,19 @@
 import { arbitrumChainId } from '@shapeshiftoss/caip'
 import type { Asset } from '@shapeshiftoss/types'
+import uniqBy from 'lodash/uniqBy'
 
 import { arbitrum } from '../baseAssets'
 import * as coingecko from '../coingecko'
 import { getRenderedIdenticonBase64 } from '../generateAssetIcon/generateAssetIcon'
+import { getPortalTokens } from '../utils/portals'
 
 export const getAssets = async (): Promise<Asset[]> => {
-  const assets = await coingecko.getAssets(arbitrumChainId)
-  return [...assets, arbitrum].map(asset => ({
+  const [assets, portalsAssets] = await Promise.all([
+    coingecko.getAssets(arbitrumChainId),
+    getPortalTokens(arbitrum),
```

Comment:
I'm assuming we are happy stopping here if one of those calls fail? I.e. we are happy with `Promise.all` vs `Promise.allSettled`.

Reply:
Absolutely in on that one - the reason why it "worked" for Ethereum (which uses `Promise.all()` is Idle throws and doesn't recatch, and the two other Uni-V2/Coingecko asset services "normally work", which as we can probably all agree is definitely not safe 😄 

d59b7f72f9aa3b11b02856547d4bfaef76d0f9